mico-core/src/main/java/io/github/ust/mico/core/dto/CrawlingInfoDTO.java

```diff
@@ -70,4 +74,21 @@
             @ExtensionProperty(name = "description", value = "The remote release tag. Defaults to 'latest'.")})})
     @NotEmpty
     private String version = "latest";
+
+
+    /**
+     * The path to the dockerfile must be relative to the root folder of the git repository
```

Comment:
To be consistent `dockerfile` should be written with a uppercase D → `Dockerfile` (don't forget to adjust the `ExtensionProperty`)

Reply:
Done